webrender/src/prim_store.rs

```diff
@@ -298,6 +302,7 @@ pub enum BrushClipMaskKind {
 pub struct BrushSegmentDescriptor {
     pub segments: Vec<BrushSegment>,
     pub clip_mask_kind: BrushClipMaskKind,
+    pub source_per_segment: bool,
```

Comment:
I am not sure what to call this (I am only sure that I a don't like the current name I gave it). Right now this defines how to compute the UVs for each segment (either have all segments sample the entire source image or have segments sample parts of the source image).
Suggestions are very much welcome.

Maybe an enum `SegmentUv` with variants `Auto` and `Full` would be better than a boolean.